saa/4.1. Compute - EC2.md

```diff
@@ -179,7 +195,7 @@
 - **Spot instances**
   - Cheapest (up to 90% discount compared to on demand) but risk of losing instance
     - Spot instance is charged per second.
-    - If you get shut down in first hour you don't pay.
+    - If aws shut down your instance in first hour you don't pay, but you have to pay if you terminate.
```

Comment:
Very nice addition. One small thing: aws should be AWS (capitalized)

Reply:
Thanks.